lib/index.js

```diff
@@ -76,6 +77,9 @@ class PWMetrics {
     if (this.opts.json) {
       return data;
     } else if (this.opts.expectations) {
+      if ( this.opts.showChart && 'false' != this.opts.showChart ) {
```

Comment:
Not sure if this is the best way to do this, however it seems that setting `--showChart=false` in CLI passes the `showChart` option through as a string instead of a bool. I thought about automatically parsing this in `cli.js`, however I think this is a different task altogether, and it seems like there are plenty of packages out there to handle this if you decide to go that route. Would love to hear if you have a better suggestion on how to handle this more gracefully.